monkey/tests/unit_tests/infection_monkey/ransomware/test_bitflip_encryptor.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+from tests.unit_tests.infection_monkey.ransomware.ransomware_target_files import (
+    TEST_KEYBOARD_TXT,
+    TEST_KEYBOARD_TXT_CLEARTEXT_SHA256,
+    TEST_KEYBOARD_TXT_ENCRYPTED_SHA256,
+)
+from tests.utils import hash_file
+
+from infection_monkey.ransomware.bitflip_encryptor import BitflipEncryptor
+
+
+def test_file_encrypted(ransomware_target):
+    test_keyboard = ransomware_target / TEST_KEYBOARD_TXT
+
+    assert hash_file(test_keyboard) == TEST_KEYBOARD_TXT_CLEARTEXT_SHA256
+
+    encryptor = BitflipEncryptor(chunk_size=64)
```

Comment:
Why is the chunk size in unit tests different from the default chunk size?

Reply:
Since the encryption routine reads files in chunks, we want to test that encryption occurs correctly if a chunk is smaller than `chunk_size`. To that end, the size of `test_keyboard.txt` was deliberately chosen to be less than `chunk_size * 2` but greater than `chunk_size * 1`. The test will read a full chunk and encrypt it, followed by a partial chunk. By specifying the size as 64 in the unit test, we're ensuring that regardless of how the chunk size changes throughout the rest of the code, these tests will continue to test encryption of partial chunks.